packages/manager/modules/server-sidebar/src/controller.js

```diff
@@ -148,6 +149,9 @@ export default class OvhManagerServerSidebarController {
 
   filterFeatures(items) {
     return filter(items, (item) => {
+      if (has(item, 'children')) {
+        set(item, 'children', this.filterFeatures(item.children));
+      }
```

Comment:
```suggestion
      if (item.children) {
        item.children = this.filterFeatures(item.children);
      }
```

Reply:
If I remove the lodash, I have a message saying that it is one of the function's params.